### PR TITLE
53 minor model issues

### PR DESCRIPTION
**Pull request by ricco386**
I have updated missing stuff in issue #53 please review...

### CODE REVIEW

File: konfera/admin.py [hunk 1]
```diff
@@ -34,8 +34,6 @@ class EventAdmin(admin.ModelAdmin):
         'slug': ('title',),
     }
 
-admin.site.register(Event, EventAdmin)
```
Comment (ZoltanOnody) [inline konfera/admin.py]: Why did you delete this?

File: konfera/migrations/0001_initial.py [hunk 2]
```diff
@@ -31,14 +31,17 @@ class Migration(migrations.Migration):
...
+                ('date_from', models.DateTimeField(verbose_name='Begining')),
+                ('date_to', models.DateTimeField(verbose_name='End')),
                 ('title', models.CharField(max_length=128, verbose_name='Event name')),
                 ('slug', models.SlugField(help_text='Slug field, relative URL to the event.', verbose_name='Event url')),
                 ('description', models.TextField()),
-                ('date_from', models.DateTimeField(verbose_name='Event begging')),
-                ('date_to', models.DateTimeField(verbose_name='Event end')),
-                ('event_type', models.CharField(choices=[('conference', 'Conference'), ('meetup', 'Meetup')], max_length=20)),
-                ('status', models.CharField(choices=[('draft', 'Draft'), ('published', 'Published'), ('expired', 'Expired')], max_length=20)),
+                ('event_type', models.CharField(choices=[(b'conference', 'Conference'), (b'meetup', 'Meetup')], max_length=20)),
```
Comment (ZoltanOnody) [inline konfera/migrations/0001_initial.py]: `b'conference'` seems like you generated the migrations in Python 2

File: konfera/models/schedule.py [hunk 3]
```diff
@@ -1,11 +1,20 @@
+from django.core.validators import MinValueValidator, MaxValueValidator
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
 
 class Schedule(models.Model):
     start = models.DateTimeField()
-    description = models.TextField()
+    description = models.TextField(
+        help_text=_('Description will be displayed, only if there is no related talk, eg. coffee break, lunch etc...'))
```
Comment (ZoltanOnody) [inline konfera/models/schedule.py]: > Description will be displayed, **only if there is no related talk**

so the field should be probably optional

File: konfera/models/abstract.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+class FromToModel(models.Model):
+
+    class Meta:
+        abstract = True
+
+    date_from = models.DateTimeField(verbose_name=_('Begining'))
+    date_to = models.DateTimeField(verbose_name=_('End'))
+
+    def clean(self):
+        if self.date_from > self.date_to:
```
Comment (ZoltanOnody) [inline konfera/models/abstract.py]: Not important, but it's should be >= to match the error message.

File: konfera/models/talk.py [hunk 5]
```diff
@@ -23,13 +24,20 @@
 
 class Talk(models.Model):
     title = models.CharField(max_length=256)
-    abstract = models.TextField()
+    abstract = models.TextField(help_text=_('Abstract will be published in the schedule.'))
     type = models.CharField(choices=TALK_TYPE, max_length=32)
     status = models.CharField(choices=TALK_STATUS, max_length=32)
-    duration = models.IntegerField(choices=TALK_DURATION)
+    duration = models.IntegerField(choices=TALK_DURATION, help_text=_('Talk duration in minutes.'))
     primary_speaker = models.ForeignKey('Speaker', related_name='secondary_speaker')
```
Comment (ZoltanOnody) [inline konfera/models/talk.py]: (you didn't change this line but) I noticed that `primary_speaker` has related_name `secondary_speaker` an vice versa.

File: konfera/models/talk.py [hunk 6]
```diff
@@ -23,13 +24,20 @@
...
     secondary_speaker = models.ForeignKey('Speaker', related_name='primary_speaker', blank=True, null=True)
     event = models.ForeignKey('Event')
 
     def __str__(self):
         return self.title
+
+    def clean(self):
+        if self.primary_speaker == self.secondary_speaker:
```
Comment (ZoltanOnody) [inline konfera/models/talk.py]: This validation doesn't seem to work as supposed.

If you don't select the primary speaker it throws `RelatedObjectDoesNotExist at /admin/konfera/talk/add/
Talk has no primary_speaker.` instead of validation error.

File: konfera/admin.py (hunk 1 above)
Comment (ricco386) [inline konfera/admin.py]: I am using 2 computers and somehow it get lost...

File: konfera/migrations/0001_initial.py (hunk 2 above)
Comment (ricco386) [inline konfera/migrations/0001_initial.py]: unfortunately yes, I am using 2 computers and on the other one is python2 default...

File: konfera/models/schedule.py (hunk 3 above)
Comment (ricco386) [inline konfera/models/schedule.py]: yes, you are right as well as room and talk. Thanks for spotting this.

File: konfera/models/talk.py (hunk 6 above)
Comment (ricco386) [inline konfera/models/talk.py]: I havent tested properly, thanks for spotting.

**Comment by ricco386** [comment]
@ZoltanOnody please review the fixes and I will merge once approved. Thanks.

File: konfera/models/abstract.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+class FromToModel(models.Model):
+
+    class Meta:
+        abstract = True
+
+    date_from = models.DateTimeField(verbose_name=_('Beginning'))
+    date_to = models.DateTimeField(verbose_name=_('End'))
+
+    def clean(self):
+        if self.date_from >= self.date_to:
```
Comment (ZoltanOnody) [inline konfera/models/abstract.py]: `TypeError at /admin/konfera/event/add/
unorderable types: NoneType() >= NoneType()` when a blank form is submitted

File: konfera/models/abstract.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+            msg = _('%(date_from)s have to be before %(date_to)s') % {
```
Comment (ZoltanOnody) [inline konfera/models/abstract.py]: Add a period to the end of message to keep the error messages consistent.

File: konfera/models/talk.py [hunk 9]
```diff
@@ -23,13 +24,22 @@
...
+    secondary_speaker = models.ForeignKey('Speaker', related_name='secondary_speaker_talks', blank=True, null=True)
     event = models.ForeignKey('Event')
 
     def __str__(self):
         return self.title
+
+    def clean(self):
+        if hasattr(self, 'primary_speaker') and hasattr(self, 'secondary_speaker') \
+                and self.primary_speaker == self.secondary_speaker:
+            msg = _('%(primary)s have to be different than %(secondary)s') % {
```
Comment (ZoltanOnody) [inline konfera/models/talk.py]: "primary speaker have to be different than secondary speaker"

- add a period to the end
- capitalize the **P**rimary speaker
